engine/config/engine.go

```diff
@@ -87,6 +88,7 @@ type EngineConfig struct {
 	SocketPath              string        // The path to the engine socket.
 	StatsInterval           time.Duration // The statistics update interval.
 	SyncPort                int           // The port for sync'ing with this node's peer.
+	UseVMAC                 bool          // Use VRRP MAC. If not, ClusterVIP is optional and Seesaw uses gratuitous arp/unsolicited na for failover.
```

Comment:
If not => If false ? If false, ClusterVIP is not taking affect, is it? 
Also worth mentioning default is true.

Reply:
Fixed. I wrote it before I do the implementation and realized it's hard to take away clusterVIP.